lib/ui/painting/picture.cc

```diff
@@ -155,8 +155,8 @@ Dart_Handle Picture::RasterizeToImage(const sk_sp<DisplayList>& display_list,
 
   // We can't create an image on this task runner because we don't have a
   // graphics context. Even if we did, it would be slow anyway. Also, this
-  // thread owns the sole reference to the layer tree. So we flatten the layer
-  // tree into a picture and use that as the thread transport mechanism.
+  // thread owns the sole reference to the layer tree. So we do the heavy
```

Comment:
This comment was wrong but I don't like the update. Its not a heavy task and we don't do it in the raster thread to reduce the workload, we do it so that the layer tree has access to the external texture registry

Reply:
I see, thanks for the clarification and I updated it!